src/init.cpp

```diff
@@ -479,7 +479,12 @@ void SetupServerArgs(NodeContext& node)
 #endif
 #else
     hidden_args.emplace_back("-upnp");
-#endif
+#endif // #ifdef USE_UPNP
+#ifdef USE_NATPMP
+    argsman.AddArg("-natpmp", strprintf("Use NAT-PMP to map the listening port (default: %s)", DEFAULT_NATPMP ? "1 when listening and no -proxy" : "0"), ArgsManager::ALLOW_ANY, OptionsCategory::CONNECTION);
```

Comment:
Nit: linebreak would be nice.

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/18077#issuecomment-714683741).

Reply:
Reverted back as a linebreak breaks `test/lint/check-doc.py` linter: https://travis-ci.org/github/bitcoin/bitcoin/jobs/738110074